.travis.yml

```diff
@@ -75,5 +75,5 @@ deploy:
   script: ./deploy.sh
   skip_cleanup: true
   on:
-    all_branches: true
+    branch: master
```

Comment:
@sturmf 
I suppose this would skip the deploy step except for builds on master.
But would we have the same deploy error as before as soon as we merge on master and travis deploys from there?